testing/export/export-verify-explicit-patch-version-value-of-zero.cmake

```diff
@@ -59,7 +59,7 @@ if(NOT "${TEST_VERSION_MAJOR}.${TEST_VERSION_MINOR}" VERSION_EQUAL 21.09)
   message(FATAL_ERROR "rapids_export failed to export version major/minor information")
 endif()
 
-find_package(test 21.9.0 EXACT REQUIRED)
+find_package(test 21.09.0 EXACT REQUIRED)
```

Comment:
Was this previously just a typo, or did something actually change?

Reply:
This is on purpose. EXACT matches requires string equivalant so the value needs to be `.9`

Reply:
Wait now I"m confused. You changed it from `.9` to `.09`. Which should it be?

Reply:
The logic around version generated by `rapids-cmake/export/detail/parse_version.cmake` has changed. We used to always drop leading zeroes from version values as it would break cmake.

Now since that is supported we export the version value as specified by the project ( 21.09.0 ) and as a result the `EXACT` query here needs to be the same.